src/docs/guide/interactivity.md

```diff
@@ -7,4 +7,4 @@ order: 7
 show_guide: true
 ---
 
-Coming soon.
+Coming soon. See the [cursor component docs](../components/cursor.html) in the meantime.
```

Comment:
personally, I like backticking because it makes it very scannable that "oh, hey, that's the attribute name `material`, right" - wonder what others think. but I'm not married to it either way.